.github/workflows/pr-checks.yml

```
@@ -15,14 +15,41 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 45
 
+    strategy:
+      matrix:
+        node-types-version: [16.11, current] # we backport this matrix job in order to maintain the same check names
+
     steps:
       - name: Checkout
         uses: actions/checkout@v4
 
       - name: Lint
         run: npm run-script lint
 
+      - name: Update version of @types/node
+        if: matrix.node-types-version != 'current'
+        env:
+          NODE_TYPES_VERSION: ${{ matrix.node-types-version }}
+        run: |
+          # Export `NODE_TYPES_VERSION` so it's available to jq
+          export NODE_TYPES_VERSION="${NODE_TYPES_VERSION}"
```

I didn't know this was necessary.

Pretty sure this works the same:

```suggestion
          export NODE_TYPES_VERSION
```